mlir/utils/jenkins/Jenkinsfile

```diff
@@ -410,14 +419,14 @@ pipeline {
                     steps {
                         sh 'rm -rf MIOpen'
                         unstash name: "MIOpen-MLIR-kernels"
-                        //copyArtifacts filter: 'MIOpen/build/MIOpenUserDB,MIOpen/build/tuning-date',\
-                        //                       optional: true,\
-                        //                       flatten: true,\
-                        //                       projectName: "/${JOB_NAME}",\
-                        //                       selector: lastSuccessful(),\
-                        //                       target: 'MIOpen/build/'
-                        //sh 'ls MIOpen/build/MIOpenUserDB'
-                        //sh 'cat MIOpen/build/tuning-date'
+                        copyArtifacts filter: 'MIOpen/build/MIOpenUserDB/**',\
+                                               optional: true,\
+                                               flatten: true,\
+                                               projectName: "mlir-weekly",\
```

Comment:
Just curious: You already have the weekly job setup? Mind amend the confluence page to remind the new job?

Reply:
Yes. I have to. The weekly results are needed for nightlies.
 Will update the confluence page.